storybookjs/aem
# Enhancement/clean up eslint errors

Issue:
ESLint errors that could not be fixed by --fix flag

## What I did
Clean up the obvious issues and add ignore flags for eslint issues that might not be fixable.

## How to test
Run yarn build and demo to ensure all is working as expected

- Does this need an update to the documentation?
  - Not yet, will be needed when husky is added (fast follow to this pr)
